proto/viam/service/motion/v1/motion.proto

```diff
@@ -81,6 +81,23 @@ message MoveOnMapResponse {
   bool success = 1;
 }
 
+message MotionConfiguration {
+  // The name of the vision service(s) that will be used to detect obstacles
+  repeated common.v1.ResourceName vision_services = 1;
+  // Sets the frequency to poll for the position of the robot
+  optional double position_polling_frequency = 2;
+  // Sets the frequency to poll the vision service(s) for new obstacles
+  optional double obstacle_polling_frequency = 3;
+  // Sets the distance in meters that a robot is allowed to deviate from the motion plan
+  optional double plan_deviation_m = 4;
+  // If this value is set, it will set the radius of an ellipse that a robot should search in when attempting to replan.
+  optional double replan_cost_factor = 5;
```

Comment:
This is no longer going into the API.  We updated the scope doc but I forgot to let you know about it

Reply:
will remove right now